secrethub.i

```diff
@@ -32,7 +32,9 @@
 // Map return value of ResolveEnv to Dictionary<string, string>.
 %typemap(cstype) char* ResolveEnv "System.Collections.Generic.Dictionary<string,string>"
 %typemap(csout, excode=SWIGEXCODE) char* ResolveEnv {
-    var res = Newtonsoft.Json.JsonConvert.DeserializeObject<System.Collections.Generic.Dictionary<string, string>>($imcall);
+    var temp = $imcall;
```

Comment:
Can we maybe think of a more descriptive name than `temp`?

Reply:
Good point :+1: Renamed it to `jsonEnv`.